cf-check/lmdump.c

```diff
@@ -115,14 +115,16 @@ int lmdump(lmdump_mode mode, const char *file)
     if (rc != MDB_NOTFOUND)
     {
         // At this point, not found is expected, anything else is an error
-        return lmdump_report_error(rc);
+        goto cleanup;
     }
-    mdb_cursor_close(cursor);
-    mdb_close(env, dbi);
-
-    mdb_txn_abort(txn);
-    mdb_env_close(env);
+    rc = 0;
+cleanup:
+    if (cursor) mdb_cursor_close(cursor);
```

Comment:
Please use explicit comparisons and curly braces on separate lines. See [CONTRIBUTING.md](https://github.com/cfengine/core/blob/master/CONTRIBUTING.md).